### PR TITLE
Rebuild example android and ios dirs

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

**Comment by domesticmouse** [comment]
PTAL @RedBrogdon, @KagurazakaHanabi

File: example/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,20 +1,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.flutter.packages.flutter_markdown_example">
+    package="com.example.example">
```
Comment (KagurazakaHanabi) [inline example/android/app/src/main/AndroidManifest.xml]: Keep package name is io.flutter.packages.flutter_markdown_example

File: example/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
     <!-- io.flutter.app.FlutterApplication is an android.app.Application that
          calls FlutterMain.startInitialization(this); in its onCreate method.
          In most cases you can leave this as-is, but you if you want to provide
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:icon="@mipmap/ic_launcher"
-        android:label="flutter_markdown_example">
+        android:label="example"
```
Comment (KagurazakaHanabi) [inline example/android/app/src/main/AndroidManifest.xml]: Keep app name is flutter_markdown_example

File: example/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (domesticmouse) [reply example/android/app/src/main/AndroidManifest.xml]: Hopefully I successfully reverse engineered all the places I needed to patch.

File: example/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (domesticmouse) [reply example/android/app/src/main/AndroidManifest.xml]: Done

**Pull request merged**